src/edu/cs505/g5/OffState.java

```diff
@@ -0,0 +1,16 @@
+package edu.ccsu.cs505.g5;

```

Comment:
You have the package as edu.ccsu.cs505.g5, but your code is in the directory edu.cs505.g5 thus compile error

Reply:
modified